drivers/hid/hidclass/fdo.c

```diff
@@ -1,28 +1,32 @@
 /*
  * PROJECT:     ReactOS Universal Serial Bus Human Interface Device Driver
- * LICENSE:     GPL - See COPYING in the top level directory
- * FILE:        drivers/hid/hidclass/fdo.c
+ * LICENSE:     GPL-3.0-or-later (https://spdx.org/licenses/GPL-3.0-or-later)
  * PURPOSE:     HID Class Driver
- * PROGRAMMERS:
- *              Michael Martin (michael.martin@reactos.org)
- *              Johannes Anderwald (johannes.anderwald@reactos.org)
+ * COPYRIGHT:   Copyright  Michael Martin <michael.martin@reactos.org>
+ *              Copyright  Johannes Anderwald <johannes.anderwald@reactos.org>
+ *              Copyright 2022 Roman Masanin <36927roma@gmail.com>
  */
 
 #include "precomp.h"
 
 #define NDEBUG
 #include <debug.h>
 
+IO_COMPLETION_ROUTINE HidClassFDO_QueryCapabilitiesCompletionRoutine;
+IO_COMPLETION_ROUTINE HidClassFDO_DispatchRequestSynchronousCompletion;
+IO_COMPLETION_ROUTINE HidClassFDO_ReadCompletion;
+
 NTSTATUS
 NTAPI
 HidClassFDO_QueryCapabilitiesCompletionRoutine(
     IN PDEVICE_OBJECT DeviceObject,
     IN PIRP Irp,
     IN PVOID Context)
 {
-    //
-    // set event
-    //
+    UNREFERENCED_PARAMETER(DeviceObject);
+    UNREFERENCED_PARAMETER(Irp);
```

Comment:
Please, remove all the `UNREFERENCED_PARAMETER` changes. They are not relevant (and not needed, actually)

Reply:
but why? this args will not be used, because this routine only exists to change state of event

Reply:
Because this change is not relevant to the PR. Also, we usually don't put `UNREFERENCED_PARAMETER` in our code

Reply:
Agreed. Keep them! Drivers are supposed to compile with /W4